confirmed-change.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Ally seamlessly integrates with the Learning Management System and automatically generates alternative accessible formats, provides instructor feedback and generates an institutional accessibility report">
+    <meta name="author" content="Fronteer LTD">
+    <link rel="icon" href="favicon.ico">
+
+    <title>Ally - Making course content accessible</title>
+
+    <link href="//fonts.googleapis.com/css?family=Roboto:400,300,500" rel="stylesheet" type="text/css">
+    <link href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet" type="text/css">
+    <link href="assets/css/style.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+
+    <section id="top" class="triangle clearfix">
+        <!-- NAVIGATION -->
+        <nav class="clearfix">
+            <div class="hidden-xs col-sm-3">
+                <h1>
+                    <figure>
+                        <img src="/assets/img/logo-brand-mark.svg" alt="Ally" title="Ally" />
+                    </figure>
+                </h1>
+            </div>
+            <div class="col-xs-12 col-sm-9">
+                <ul>
+                    <li><a href="index.html#request-demo">Request demo</a></li>
+                    <li><a href="index.html#features">Features</a></li>
+                    <!-- <li><a href="#get-in-touch">Contact</a></li> -->
+                </ul>
+            </div>
+        </nav>
+
+        <!-- THANK YOU -->
+        <div id="thank-you">
+            <figure id="illustration">
+                <img src="assets/img/confirm.svg" alt="" title="Ally User Group" >
+            </figure>
+            <div id="title-container">
+                <h2>Thats been updated, thank you.</h2>
+            </div>
+        </div>
+
+    </section>
+
+    <div id="share" class="triangle">
+        <div id="message">
+            <p>Your contribution in the Ally User Group is important to many people, both with and without disabilities. Only with your perspective, experience and opinion, services like Ally that aim for wide and resounding impact, can continue to meaningfully contribute value.</p>
```

Comment:
"Your contributions to the Ally User Group are important to help shape the future of Ally. Let us know how we can make the Ally User Group more relevant and interesting to you at info@ally.ac."